src/stylesheets/components/_project.scss

```diff
@@ -1,3 +1,40 @@
 .project {
-  
+  max-width: 340px;
+  position: relative;
+
+  &__link {
+    display: block;
+
+    &:hover .project__hover {
+      opacity: 1;
+    }
+  }
+
+  &__image {
+    display: block;
+    width: 100%;
+  }
+
+  &__hover {
+    position: absolute;
+    top: 0; bottom: 0; left: 0; right: 0;
+    background-color: rgba(0, 0, 0, .6);
+    font-weight: 600;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```

Comment:
As always, what do you want to get with this?

Reply:
This is good for aligning multiple elements vertically

Reply:
But they are vertical from the most beginning. Why not to use `paddings` and `margin: 0 auto;`?